### PR TITLE
Support pre-processing of Gatherer data

### PR DESCRIPTION
**Pull request by ZeldaZach**
Also moves outputs to Git LFS to avoid repo pollution

### CODE REVIEW

File: gatherer_provider.py [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    "Infinite": "∞",
+}
+
+LANGUAGE_MAP: Dict[str, str] = {
+    "en": "English",
+    "es": "Spanish",
+    "fr": "French",
+    "de": "German",
+    "it": "Italian",
+    "pt": "Portuguese (Brazil)",
```
Comment (kodawah) [inline gatherer_provider.py:89]: in ISO, `pt` is for "Portuguese (Portugal)", "Portuguese (Brazil)" would be `pt-br`, is this something that should be noted?

File: outputs/.gitattributes [hunk 2]
```diff
@@ -0,0 +1,5 @@
+gatherer_mapping.json filter=lfs diff=lfs merge=lfs -text
```
Comment (kodawah) [inline outputs/.gitattributes:1]: introducing this file OR adding the other maps should be a separate commit

Comment (ZeldaZach) [reply outputs/.gitattributes:1]: Fair enough

File: gatherer_provider.py (hunk 1 above)
Comment (ZeldaZach) [reply gatherer_provider.py:89]: This is copy/paste from the main repo, and is a Gatherer thing.

Comment (kodawah) [reply gatherer_provider.py:89]: does it need to "(brazil)"? couldn't it be just Portuguese?

Comment (ZeldaZach) [reply gatherer_provider.py:89]: This is a connotation based on how Gatherer handles things, so unfortunately it's correct as-is

**Review by kodawah — APPROVED** [review]
+651 −486,653 ✅
